src/Sylius/Behat/Context/Ui/Shop/ProductContext.php

```diff
@@ -111,4 +141,53 @@ public function iShouldSeeTheProductAttributeWithValue($attributeName, $Attribut
             sprintf('Product should have attribute %s with value %s, but it does not.', $attributeName, $AttributeValue)
         );
     }
+
+    /**
+     * @Given /^I want to see products in (channel "([^"]*)")$/
```

Comment:
This step will be used in pretty much every scenario that considers more than 1 channel. So it is better to have single step context than duplicated step in all contexts.